src/core/sys/freebsd/netinet/in_.d

```diff
@@ -0,0 +1,581 @@
+//Written in the D programming language
```

Comment:
Yes, Walter insists on this whenever he happens to catch a PR with a new module addition.
I don't think it makes any difference in Google optimization nowadays, but it definitely doesn't hurt.